operator/v1/types.go

```diff
@@ -40,6 +41,14 @@ type ClusterManagerSpec struct {
 	// +optional
 	// +kubebuilder:default=quay.io/open-cluster-management/placement
 	PlacementImagePullSpec string `json:"placementImagePullSpec,omitempty"`
+
+	// NodeSelector defines which Nodes the Pods are scheduled on. The default is an empty list.
+	// +optional
```

Comment:
put NodeSelector and Tolerations in a struct. And why it is not labelSelector？

Reply:
done. put them in NodePlacement